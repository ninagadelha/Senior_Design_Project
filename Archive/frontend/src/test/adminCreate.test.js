import renderer from "react-test-renderer";
import AdminCreate from "../routes/adminCreate";

jest.mock("../components/AdminNavBar");
jest.mock("../components/footer");
jest.mock("../styles/adminResults.css");
jest.mock("react-router-dom");
jest.mock("axios");

const renderTree = (tree) => renderer.create(tree);
describe("<AdminCreate>", () => {
  it("should render component", () => {
    expect(renderTree(<AdminCreate />).toJSON()).toMatchSnapshot();
  });
});
