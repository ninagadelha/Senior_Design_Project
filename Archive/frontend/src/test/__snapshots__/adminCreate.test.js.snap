// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminCreate> should render component 1`] = `
<div
  class="cont"
>
  <div
    class="content-placeholder-results"
  >
    <h1>
      Create New Program Groups
    </h1>
    <div>
      <label>
        Enter New Program Name: 
        <input
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
      <button
        className="action-button-results"
        onClick={[Function]}
      >
        Create New Program
      </button>
    </div>
  </div>
</div>
`;
